extensions/notebook/package.json

```diff
@@ -353,7 +353,7 @@
         {
           "id": "books-explorer",
           "title": "Jupyter Books",
-          "icon": "resources/dark/open_notebook_inverse.svg"
+          "icon": "resources/dark/JupyterBook_2.svg"
```

Comment:
Just check in the one unless we actually need the other. No reason to add more clutter unless we need it.